R/classicProcess.R

```diff
@@ -1742,6 +1745,51 @@ ClassicProcess <- function(jaspResults, dataset = NULL, options) {
   return(modProbes)
 }
 
+.procPathMediationEffectsTableHelper <- function(tbl, medEffects) {
```

Comment:
this function takes mediation effects data and organizes it into a structured table format, preparing it for further analysis or visualization. How was this done before the addition of this function?

Reply:
Before this was done in the `.procPathMediationEffectsTable` function itself, but now the Bayesian analysis is also using the same routine, so I put it into a separate function.